hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/bucket/HoodieSimpleBucketIndex.java

```diff
@@ -52,10 +54,20 @@ private Map<Integer, HoodieRecordLocation> loadPartitionBucketIdFileIdMapping(
     Map<Integer, HoodieRecordLocation> bucketIdToFileIdMapping = new HashMap<>();
     hoodieTable.getMetaClient().reloadActiveTimeline();
     HoodieIndexUtils
-        .getLatestBaseFilesForPartition(partition, hoodieTable)
-        .forEach(file -> {
-          String fileId = file.getFileId();
-          String commitTime = file.getCommitTime();
+        .getLatestFileSlicesForPartition(partition, hoodieTable)
+        .forEach(fileSlice -> {
+          String fileId;
+          String commitTime;
```

Comment:
Does
```java
    fileSlice.getFileId();
    fileSlice.getBaseInstantTime();
```

work here ?

Reply:
Of course, the test code could cover this.

Reply:
@danny0405 To cow table not have log file, seems we need to consider base file. But to mor table when the log file in not compacted to base file, which we need to consider log file.

Reply:
Every file slice has basic props like fileGroupId and baseCommitTime, there is no need to look into the details if you want  only these two.

Reply:
Great ~ Hi @danny0405 , please see again, I have corrected.